frontend/src/pages/Profile.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { user } from '../reducers/user'
+import { logout } from '../reducers/user'
+import { Headline } from '../lib/headline'
+import { InputButton, ProfileImg } from '../lib/button'
+import { ProfileMessage, ProfileDiv, ProfileInfo } from '../lib/form'
+import { useDispatch, useSelector } from 'react-redux'
+import { Link } from 'react-router-dom'
+
+export const Profile = () => {
+  const dispatch = useDispatch()
+  const secretMessage = useSelector((store) => store.user.login.secretMessage)
+  const userName = useSelector((store) => store.user.login.userName)
+
+  return (
+    <ProfileInfo>
+      <Headline title='profile' />
+      <ProfileDiv>
+        {secretMessage && <ProfileMessage> {secretMessage} </ProfileMessage>}
```

Comment:
This doesn't show, because you're not setting this `secretMessage` in your reducer. From the project brief:
"Your frontend needs:
- A page to show the authenticated content from the API.

"